### PR TITLE
New unit tests for GeoConvHelper

### PR DESCRIPTION
**Pull request by vijay-jaisankar**
These are the main features of this PR

- Unit test for the `cartesian2geo` method

### CODE REVIEW

**Comment by angelobanse** [comment]
Thanks for this PR. I'm not really sure if we really use the unittests - @behrisch?

@vijay-jaisankar: If you want to help us with tests, take a look at the open issues marked with the label ["test needed"](https://github.com/eclipse/sumo/issues?q=is%3Aopen+is%3Aissue+label%3A%22test+needed%22). We use texttest ([documentation about tests](https://sumo.dlr.de/docs/Developer/Tests.html)) and track the status of tests [here](https://sumo.dlr.de/daily/).

**Comment by vijay-jaisankar** [comment]
Hello, 

The [changes](https://github.com/eclipse/sumo/pull/13675#issuecomment-1697752395) have been made in [this commit](https://github.com/eclipse/sumo/pull/13675/commits/1699ceb0ae05f82a035a08c2ef996aebbc154f98), please check.

File: unittest/src/utils/geom/GeoConvHelperTest.cpp [hunk 1]
```diff
@@ -40,3 +40,18 @@ TEST(GeoConvHelper, test_method_x2cartesian) {
...
     EXPECT_NEAR(5809666.826070101, pos.y(), 1e-5);
 }
+
+/* Test the method 'cartesian2geo' */
+TEST(GeoConvHelper, test_method_cartesian2geo) {
+    GeoConvHelper gch(
+        "+proj=utm +zone=33 +ellps=WGS84 +datum=WGS84 +units=m +no_defs",
+        Position(), Boundary(), Boundary(), 1, false);
+
+    // Use the outputs of the `x2catesian` test hence making the combination of tests circular: geo -> cartesian -> geo
```
Comment (behrisch) [inline unittest/src/utils/geom/GeoConvHelperTest.cpp:50]: Please fix the typo in 'x2cartesian'

**Comment by behrisch** [comment]
Thank you!

**Pull request merged**